elements/pl-symbolic-input/pl-symbolic-input.py

```diff
@@ -45,7 +46,13 @@ def render(element_html, data):
         operators = ', '.join(['cos', 'sin', 'tan', 'exp', 'log', 'sqrt', '( )', '+', '-', '*', '/', '^', '**'])
         constants = ', '.join(['pi'])
 
-        info_params = {'format': True, 'variables': variables_string, 'operators': operators, 'constants': constants}
+        info_params = {
+            'format': True,
+            'variables': variables_string,
+            'operators': operators,
+            'constants': constants,
+            'allow_complex': allow_complex,
+        }
```

Comment:
Do we explicitly check somewhere that the instructor has not asked for `pi` or `e` to be a variable?

Reply:
We check for conflicts, yep.